lib/async-base-container.js

```diff
@@ -0,0 +1,125 @@
+/** @format */
+
+import config from 'config';
+import assert from 'assert';
+import { isEqual } from 'lodash';
+
+import * as driverManager from './driver-manager';
+import * as driverHelper from './driver-helper';
+import * as slackNotifier from './slack-notifier';
+
+export default class AsyncBaseContainer {
+	constructor( driver, expectedElementSelector, url = null ) {
+		this.name = this.constructor.name;
+		this.driver = driver;
+		this.screenSize = driverManager.currentScreenSize().toUpperCase();
+		this.expectedElementSelector = expectedElementSelector;
+		this.url = url;
+	}
```

Comment:
I don't see that `this.explicitWaitMS` is defined somewhere. But it is used below and in other pages. Should it be added into the constructor?